File: 5.java
private final int n = 3;

private void withGoto(int[][] x)
{
	int i;
	int j;
	for (i = 0; i < n; i++)
	{
		for (j = 0; j < n; j++)
		{
			if (x[i][j] != 0)
			{
				goto reject;
			}
		}
		System.out.printf("First all zero row is: %d", i);
		break;
reject:
		;
	}
}

private void withOutGoto(int[][] x)
{
	boolean flag = false;
	int i;
	int j;
	for (i = 0; i < n; i++)
	{
		int colCounter = 0;
		for (j = 0; j < n; j++)
		{
			if (x[i][j] == 0)
			{
				colCounter++;
			}
		}
		if (flag == false && colCounter == n)
		{
			System.out.printf("First all zero row is: %d", i);
			flag = true;
		}
	}
}

private static int Main()
{
	int[][] x =
	{
		{0, 1, 2},
		{0, 0, 0},
		{8, 9, 10}
	};
	withOutGoto(x);
	return 0;
}

File: 5.c
#include <stdio.h>
#include <conio.h>

int main(){
   int i ,j ;

   int x[3][4] = {
{0, 5, 5, 0} ,   
{4, 5, 6, 7} ,
{0, 0, 0, 0} };

for (i =0; i<= 3 ;i++)
{
   for (j=0;j<3;)
   {
   if(x[i][j]==0)
   j++;
   else
   break;
}
   if(j==3)
   {
   printf("First all Zero row is : %d ", i+1);
   break;
}
}
  
return 0;
  
}
